#!/usr/bin/env python
PACKAGE = "multiple_kinects"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("cannyLowThreshold", int_t, 0, "CannyLowThreshold",    50, 0,   200)
gen.add("cannyHighThreshold", int_t, 0, "cannyHighThreshold",    200, 0,   1000)
gen.add("cannyKernel", int_t, 0, "cannyKernel",    3, 0,   20)
gen.add("houghResolution", int_t, 0, "houghResolution",    180, 0,   1000)
gen.add("houghThreshold", int_t, 0, "houghThreshold",    50, 0,   200)
gen.add("minLinLength", int_t, 0, "minLinLength",    50, 0,   200)
gen.add("maxLineGap", int_t, 0, "maxLineGap",    10, 0,   200)

exit(gen.generate(PACKAGE, "multiple_kinects", "line_detection"))
